addons/mail/models/mail_thread.py

```diff
@@ -4828,6 +4828,10 @@ def _get_allowed_message_post_params(self):
     def _get_allowed_access_params(self):
         return set()
 
+    @api.model
+    def _get_allowed_message_update_params(self):
+        return {"scheduled_date"}
+
```

Comment:
Where place it there there and not just after `_get_allowed_message_post_params`?